### DISCUSSION TITLE
Effdet for Covid

### DISCUSSION THREAD

**Discussion by jaideep11061982** [rwightman/efficientdet-pytorch#240] [General]
@rwightman 
have u guys tried effdet for covid19 detection. I am not sure but its not working at all. mAP is not increasing and  reaching even to 0.1
I dont see any public  kernel also posting Effdet  training  in competition.

**Comment by mikeb127** [comment]
I have posted an example of it for this competition [here](https://www.kaggle.com/mikeb127/efficientdet-pytorch-training-starter)

**Reply by jaideep11061982** [reply]
I too created it problem is  mAP dsnt gets better at all.
